napari/_tests/test_mouse_bindings.py

```diff
@@ -11,7 +11,7 @@ def test_viewer_mouse_bindings(make_napari_viewer):
     """Test adding mouse bindings to the viewer"""
     np.random.seed(0)
     viewer = make_napari_viewer()
-    view = viewer.window._qt_viewer
+    view = viewer.window._qt_viewer.canvas
```

Comment:
I think that when this test starts depending on `canvas`, not `qt_viewer` then this variable should be renamed to `canvas`.

Reply:
Resolved with [db735c4](https://github.com/napari/napari/pull/5432/commits/db735c45304f62ca1bc037204b694daccbabd0c3)